modules/dists/StencilDist.chpl

```diff
@@ -331,7 +331,7 @@ class StencilArr: BaseRectangularArr {
   var dom: unmanaged StencilDom(rank, idxType, stridable, ignoreFluff);
   var locArr: [dom.dist.targetLocDom] unmanaged LocStencilArr(eltType, rank, idxType, stridable);
   pragma "local field"
-  var myLocArr: unmanaged LocStencilArr(eltType, rank, idxType, stridable);
+  var myLocArr: unmanaged LocStencilArr(eltType, rank, idxType, stridable)?;
```

Comment:
What would it take to make this non-nilable?  Is this a case where we set everything up in plain-old methods rather than StencilArr.init()?

Reply:
It's done in `setup` now,. Trying to migrate that code literally over to an initializer phase 1 would seem challenging because of the coforall/on. But, maybe there's a way to order the initialization differently to make this easier.